### ISSUE TITLE
Idea for new Video

### ISSUE THREAD

**Issue by johngrabner** [AladdinPerzon/Machine-Learning-Collection#4]
3  ideas:

1) a cheat sheet with a video on all the functions in PyTorch and what context function x is used.
2) key lessons for deep learning. Example: skip connections good, normalize data good, drop out good but when used with batch normalize ...
3) what to expected when training big networks, how does loss normally behave.

btw, your einsum video was perfect.

**Comment by AladdinPerzon** [comment]
1. This is incredibly broad and I'm afraid would take a very long time. I just made a video that took hours to make on a single function in einsum hehe :) I think in regards to this it helps to be specific.

2. I think it's a good idea to talk more about deep learning concepts, right now my videos are mostly coding, will think about this some more..

3. Most of my videos to this point are very specific on a topic, but I think it's a great suggestion you bring up to show more practical aspects of deep learning, how I train models etc (I also have much to learn here in terms of faster debugging, methods for training faster etc).

Thanks for sharing your thoughts!

**Comment by johngrabner** [comment]
My newbie perspective  makes everything look broad. Your response informs me that this will not change as I learn more. Good to know.

What about a video showing built in loss functions vs input sweep?  Some has expected inputs between 0 and 1 while others are -1 to 1. This through me for a loop.
